opentracing-kafka-client/src/test/java/io/opentracing/contrib/kafka/TracingKafkaTest.java

```diff
@@ -186,9 +222,9 @@ private void createConsumer(final CountDownLatch latch, final Integer key,
       if (withInterceptor) {
         consumer = kafkaConsumer;
       } else {
-        consumer = new TracingKafkaConsumer<>(kafkaConsumer, mockTracer);
+        consumer = (consumerNameProvider == null) ? new TracingKafkaConsumer<>(kafkaConsumer, mockTracer)
+                : new TracingKafkaConsumer<>(kafkaConsumer, mockTracer, consumerNameProvider);
```

Comment:
If the `TracingKafkaConsumer` constructor is able handle null `consumerNameProvider`, we can get rid of these checks.